userprofile/factories.py

```diff
@@ -63,18 +63,24 @@ class Meta:
     github_handle = "test-org"
 
 
-def make_user_and_organization(organization_name="Test Org", github_handle="test-org", github_username="edx"):
+def make_user_and_organization(username="testuser", org_name="Test Org", org_handle="test-org", github_username="edx"):
     """
-    Create user, userprofile and organization needed for reference on instances
+    Create user, userprofile and organization needed for reference on instances.
+
+    :param username: The username of User object
+    :param org_name: The organization name the user belongs to
+    :param org_handle: The organization github handle the user belongs to
+    :param github_username: The user's username on github
+    :return: A pair of the created userprofile and the organization that profile belongs to
```

Comment:
@ahmedaljazzar Thanks for documenting these parameters! I had some small suggestions for additional polish. Here's what the L70-L74 would look like with that polish applied:

```
    Create User, UserProfile and Organization to reference from individual instances.

    :param username: The username of the User to create.
    :param org_name: The name of the Organization to create. (The user to create will be added to that organization.)
    :param org_handle: The GitHub handle of the organization to create.
    :param github_username: The GitHub username of the user to create.
    :return: A tuple consisting of the created UserProfile and Organization.
```